site/assets/images/logo-0.1-horizontal.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
maybe just my machine, but it looks like this image cuts off the last `n`... otherwise looking beautiful 😄